### PR TITLE
Fix Tactic Motion Constraint Bug

### PR DESCRIPTION
**Pull request by jonathanlew**
### Description
There was a bug where the play was using the goalie's motion constraints for all tactics. This PR fixes this by adding the correct motion constraints for each tactic

### Testing Done
visual confirmation, make more functions static in play.h/cpp so it the mistake can't be repeated

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [ ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [ ] **Remove all commented out code**
- [ ] **Remove extra print statements**: for example, those just used for testing
- [ ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue

### CODE REVIEW

File: src/software/ai/hl/stp/play/play.h [hunk 1]
```diff
@@ -111,12 +111,17 @@ class Play
...
-                  const World& world, TacticVector tactic_vector,
-                  const std::vector<Robot> robots_to_assign);
+    std::tuple<std::vector<Robot>, std::unique_ptr<TbotsProto::PrimitiveSet>,
+               std::map<std::shared_ptr<const Tactic>,
+                        RobotId>> static assignTactics(const GlobalPathPlannerFactory&
+                                                           path_planner_factory,
+                                                       const World& world,
+                                                       TacticVector tactic_vector,
+                                                       const std::vector<Robot>
+                                                           robots_to_assign);
```
Comment (akhilveeraghanta) [inline src/software/ai/hl/stp/play/play.h]: some whacky formatting lol

**Review by akhilveeraghanta — DISMISSED** [review]
lgtm!

File: src/software/ai/hl/stp/play/play.h (hunk 1 above)
Comment (jonathanlew) [reply src/software/ai/hl/stp/play/play.h]: fixed

**Review by akhilveeraghanta — APPROVED** [review]

**Review by amrm3lm — APPROVED** [review]
lgtm

**Pull request merged**